arm-sql/2014-04-01/swagger/connectionPolicies.json

```diff
@@ -0,0 +1,275 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure SQL Server API spec",
+    "description": "The Azure SQL Server management API provides a RESTful set of web services that interact with Azure SQL Server services to manage your databases. The API enables users update server connection policy.",
+    "version": "2014-04-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/connectionPolicies/{connectionPolicyName}": {
+      "put": {
+        "tags": [
+          "ConnectionPolicies"
+        ],
+        "operationId": "Servers_CreateOrUpdateConnectionPolicy",
+        "description": "Creates or updates the server's connection policy.",
+        "x-ms-examples": {
+          "Create or update a server's secure connection policy": { "$ref": "../examples/ServerConnectionPolicyCreateOrUpdate.json" }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "name": "connectionPolicyName",
```

Comment:
Can this be moved to the top level parameters section since this definition appears to be shared in multiple places?